packages/boost-demo/readme.md

```diff
@@ -0,0 +1,9 @@
+# Boost Integration Demo
+
+This is basic demo designed to work with a [forked version of boost](https://github.com/statechannels/boost) that requires payments before serving up a file via `booster-http`.
+
+It provides some basic functionality to request a payload from `booster-http` and handles downloading the file or displaying the error(such as a 402- Payment Required). The selected payment channel will be passed into the request to `booster-http` for the payment check. A `Pay` button makes a payment on selected payment channel.
+
+It relies on a go-nitro rpc server network (which can be started using [this script](https://github.com/statechannels/go-nitro/blob/5b8c876d34638f9c322cf332bf758f5e9c284907/scripts/start-rpc-servers.go)) and a running instance of our forked version of `booster-http`.
```

Comment:
Also the lotus devnet?

Reply:
TODO: let's add some of the screengrabs and sequence diagrams from here https://www.notion.so/pl-strflt/Magmo-PL-Meeting-Notes-804ef9ae93854c589038af05e85e42f2

Reply:
#70